project/plugins.sbt

```diff
@@ -1,4 +1,5 @@
 resolvers += "HAT Library Artifacts Releases" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-releases.hubofallthings.com"
+resolvers += "HAT Library Artifacts Snapshots" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-snapshots.hubofallthings.com"
```

Comment:
we do not want the snapshot repository unless it is absolutely required, and it is not for plugins